docker-compose.yml

```diff
@@ -408,19 +415,39 @@ services:
     depends_on:
       mysql:
         condition: service_healthy
-        required: false
       kong-dev:
         condition: service_healthy
-        required: false
       kong-prod:
         condition: service_healthy
-        required: false
     profiles:
       - dev
       - prod
       - server
     <<: *obn_server_config
 
+  maildev.net:
```

Comment:
a quick setup command for those that want to use a local demo mail server is:

```
docker run -d \
  --name maildev.net \
  -p 3007:3007 \
  -p 1080:80 \
  -e MAILDEV_SMTP_PORT=${EMAIL_PORT} \
  -e MAILDEV_USER=${EMAIL_USER} \
  -e MAILDEV_PASS=${EMAIL_PASSWORD} \
  --network obn-net \
  maildev/maildev bin/maildev --web 80 --smtp 3007
  ```
  
 this can be included on the README, also a custom name has been set for the obn-net network, you might want to merge the main branch into your local branch before trying this out.